pkg/config/config.go

```diff
@@ -120,36 +138,63 @@ func (c *Config) ContractAddr() common.Address {
 	return common.HexToAddress(c.ServiceContractAddress)
 }
 
-// LoadPrivateKey loads a private key from a file
-// The file can contain either hex-encoded or raw bytes
-func LoadPrivateKey(path string) (*ecdsa.PrivateKey, error) {
-	data, err := os.ReadFile(path)
+// LoadServiceSigner loads a multibase-encoded service key string
+// and wraps it in a signer along a DID
+func LoadServiceSigner(key string, did string) (principal.Signer, error) {
```

Comment:
Let's remove the name 'Signer' from this method, would like to reserve that for the naming scheme for the key we now refer to as the 'SigningKey'. Alternatives like `LoadServiceIdentity` or `LoadServicePrincipal` would be preferred.